rapid/master/data/migrations/env.py

```diff
@@ -38,8 +38,9 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-target_metadata = db.Model.metadata
-
+target_metadata = get_declarative_base().metadata
+# print(target_metadata)
```

Comment:
you want this left in here commented out?

Reply:
Nope! 👍 Thanks.